test/scheduler/onlinetesting/ExpireOnlineTestJobSpec.scala

```diff
@@ -45,7 +45,7 @@ class ExpireOnlineTestJobSpec extends BaseServiceSpec with ShortTimeout {
     def interval: FiniteDuration = mock[FiniteDuration]
   }
 
-  "expire test phase 1 job" should {
+  "expire test phase N job" should {
```

Comment:
I can tidy the exports up, but when i put the tests in for those two events they didn't seem worth it, they weren't really testing anything. What do you think?